### PR TITLE
Docs: update observability install

### PR DESCRIPTION
**Pull request by Somefive**
Signed-off-by: Yin Da <yd219913@alibaba-inc.com>

### Description of your changes

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] Update `sidebar.js` if adding a new page.
- [x] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -179,8 +179,11 @@ module.exports = {
           label: 'Automated Observability',
           collapsed: true,
           items: [
-            'platform-engineers/operations/observability',
```
Comment (wonderflow) [inline sidebars.js:182]: we should keep the path as it's a link in other docs such as blogs or docs.

File: sidebars.js [hunk 2]
```diff
@@ -179,8 +179,11 @@ module.exports = {
...
-            'platform-engineers/operations/logging',
+            'platform-engineers/observability/quickstart',
```
Comment (wonderflow) [inline sidebars.js]: the quickstart doc should have this path : 'platform-engineers/operations/observability',

File: sidebars.js [hunk 3]
```diff
@@ -179,8 +179,11 @@ module.exports = {
           label: 'Automated Observability',
```
Comment (wonderflow) [inline sidebars.js:179]:
```suggestion
          label: 'Automated Observability',
          link: {
            type: "doc",
            id: 'platform-engineers/operations/observability',
          },
```

File: docs/platform-engineers/observability/integration.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+![grafana-apikeys](../../resources/grafana-apikeys.jpg)
+
+Then copy the token into your grafana registration configuration.
+
+![grafana-added-apikeys](../../resources/grafana-added-apikeys.jpg)
+
+After the application is successfully dispatched, you can check the registration by running the following command.
+
+```shell
+> kubectl get grafana
```
Comment (wonderflow) [inline docs/platform-engineers/observability/integration.md]:
```suggestion
kubectl get grafana
```

File: docs/platform-engineers/observability/integration.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+NAME         ENDPOINT                                                      CREDENTIAL_TYPE
```
Comment (wonderflow) [inline docs/platform-engineers/observability/integration.md]: 输出跟命令分开

File: docs/platform-engineers/observability/integration.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+Now you can manage your dashboard and datasource on your grafana instance through the native Kubernetes API as well.
+
+```shell
+# show all the dashboard you have
+kubectl get grafanadashboard -l grafana=my-grafana
+# show all the datasource you have
+kubectl get grafanadatasource -l grafana=my-grafana
+```
```
Comment (wonderflow) [inline docs/platform-engineers/observability/integration.md]: split them

File: sidebars.js (hunk 1 above)
Comment (Somefive) [reply sidebars.js:182]: ok

File: sidebars.js (hunk 2 above)
Comment (Somefive) [reply sidebars.js]: ok

File: docs/platform-engineers/observability/integration.md (hunk 4 above)
Comment (Somefive) [reply docs/platform-engineers/observability/integration.md]: ok

**Review by wonderflow — APPROVED** [review]
已有的中文文档 logging、Observability 也对应调整一下，没有中文的可以先不加中文。

**Pull request merged**